src/components/SQForm/SQForm.tsx

```diff
@@ -34,17 +64,25 @@ function SQForm({
   // HACK: This is a workaround for: https://github.com/mui-org/material-ui-pickers/issues/2112
   // Remove this reset handler when the issue is fixed.
   const handleReset = () => {
-    document && document.activeElement && document.activeElement.blur();
+    document &&
+      document.activeElement &&
+      (document.activeElement as HTMLElement).blur();
   };
 
   const handleSubmit = useDebouncedCallback(
-    (...args) => onSubmit(...args),
+    (values: Values, formikHelpers: FormikHelpers<Values>) => {
+      if (typeof onSubmit === 'undefined') {
```

Comment:
If we're keeping onSubmit being optional we could potentially do `onSubmit && onSubmit(values, formikHelpers)`, right?

Reply:
I disagree with making `onSubmit` optional. We are promoting a use of this Form component it's not really meant for. 

Formik itself makes the `onSubmit` required as well:

https://github.com/jaredpalmer/formik/blob/e677bea8181f40e6762fc7e7fb009122384500c6/packages/formik/src/types.tsx#L215

To fill the hole in our library we need to build the actual feature that would cause a use to use SQForm without supplying `onSubmit`.

https://github.com/SelectQuoteLabs/SQForm/issues/561

Reply:
Ok, sounds good @SeanGroff.  I will revert this change since this will be addressed in #561.